### PR TITLE
Create openapi.yaml in addition to the links to postman

### PR DESCRIPTION
**Pull request by pavog**
Hi, I would like to extend the documentation for the api with this openapi / swagger file that is a description for the api.
Read more about swagger and openapi [here](https://swagger.io/specification/).
This file is written for openapi 2 (swagger). I may update it to 3 in the future.
This file may help me or someone else to create a user interface for interacting with the API.

### CODE REVIEW

File: openapi.yaml [hunk 1]
```diff
@@ -0,0 +1,468 @@
...
+      - in: "body"
+        name: "body"
+        description: "The address object that needs to be added"
+        required: true
+        schema:
+          $ref: "#/definitions/Address"
+      responses:
+        "200":
+          description: "Address successfully created"
+        "405":
```
Comment (samc621) [inline openapi.yaml:39]: Did you mean 404 here? You can check `/helpers/server-response.js` to see the possible server response codes.

**Review by samc621 — CHANGES_REQUESTED** [review]
Hi @pavog thanks for this contribution, and congrats on your first to this repo! I am quite familiar with the Swagger OpenAPI specification, and think this is a great addition to this repo. 

Just one comment regarding server response codes. There was one place with `405` that appeared to be a typo. Elsewhere, we use `400` for any kind of validation error, or `500` for most other errors. Thus, any endpoint with validation can return a `400` error, and any endpoint can return a `200` or `500` error.

File: openapi.yaml (hunk 1 above)
Comment (pavog) [reply openapi.yaml]: Fixed

File: openapi.yaml [hunk 2]
```diff
@@ -0,0 +1,827 @@
...
+          type: integer
+        notification_email_address:
+          type: string
+          format: email
+    ApiResponse:
+      type: object
+      properties:
+        success:
+          type: boolean
+          format: int32
```
Comment (samc621) [inline openapi.yaml:822]: Any reason for `int32` and not `int64` here? Just for consistency across the schema.

**Review by samc621 — CHANGES_REQUESTED** [review]
@pavog thanks for the updates! Just one more inline comment and then this is good to go!

File: openapi.yaml (hunk 2 above)
Comment (pavog) [reply openapi.yaml]: Oops, that was a leftover from the previous/copied property. Removed the format there.

**Review by samc621 — APPROVED** [review]
@pavog thanks, this LGTM now!

**Pull request merged**